code/datums/objective.dm

```diff
@@ -147,13 +147,17 @@ proc/create_fluff(datum/mind/target)
 		target_name = pick(items)
 		switch(target_name)
 			if("Head of Security\'s beret")
-				steal_target = /obj/item/clothing/head/hos_hat
+				if(!countJob("Head of Security"))
+					return
```

Comment:
Hm. What happens to the objective when the early return is hit? Would perhaps be better to remove the items from the list of possible steal targets before the switch